components/locale/az_AZ.tsx

```diff
@@ -0,0 +1,117 @@
+export default {
```

Comment:
![image](https://user-images.githubusercontent.com/6134547/74260463-80a29c00-4d34-11ea-81bc-bc68cf1d50aa.png)

Please refer to this code.